bin/copy-scripts-recorder

```diff
@@ -2,11 +2,24 @@
 
 set -e
 
-cp node_modules/rrweb/dist/record/rrweb-record.min.js frontend/dist/recorder.js
-cp node_modules/rrweb/dist/record/rrweb-record.min.js.map frontend/dist/recorder.js.map
-sed -i -e s/rrweb-record.min.js.map/recorder.js.map/ frontend/dist/recorder.js
+# If recorder.js files don't exist in posthog-js/dist generate recorder.js the old way to maintain backwards compatibility.
+# `array.full.js` files are explicitly built in posthog-js starting from v1.44.0
+FILE=node_modules/posthog-js/dist/array.full.js
```

Comment:
I don't fully understand this. Why wouldn't we have `array.full.js`? 
If we are releasing the updated posthog-js first then we will definitely have this file....

Also more generally, why don't we rename this file to `copy-posthog-js` and then include all the files in one go. I think that would be clearer for future readers.